tools/content-validation/eng/pipelines/python/start.yml

```diff
@@ -0,0 +1,132 @@
+name: Run switched packages content validation for python
```

Comment:
Is this specific to python?

I assume this pipeline is expected to be created in our DevOps org. Can you please name the yml file closer to what it is about? start.yml isn't very descriptive perhaps something like docs-content-validation.yml.

Reply:
Yes, it is specific to python, in our current design, each language should have its own pipeline.

Updated to `python-sdk-docs-content-validation.yml`